packages/medusa/src/models/store.ts

```diff
@@ -54,6 +54,9 @@ export class Store extends BaseEntity {
   @Column({ nullable: true })
   invite_link_template: string
 
+  @Column({ nullable: true })
+  default_location_id: string
```

Comment:
thought (non-blocking): maybe we should also have a default location on the sales channel level. Nothing actionable right now, but worth keeping an eye out for if we start seeing a need for fallback behavior at the sales channel level.

Reply:
@srindom, I will handle all these `location_id` fields on https://github.com/medusajs/medusa/pull/2907 

@pKorsholm I think we just have to align the migrations. You could have this field in a migration and I have split the Stock location and inventory in other 2 different migrations. If you add this field in this PR I can remove it from mine.